docs/getting_started/concepts.rst

```diff
@@ -2,92 +2,134 @@
 
 Snapshot
---------
+^^^^^^^^
 
 A snapshot is a version of a ``Dataset`` containing ``annotations`` at a given time. Snapshots can be created through the Rubrix UI so they can be loaded and used using the Python library.
 
 Record
-------
+^^^^^^
 
 A record is a data item composed of ``inputs`` and, optionally, ``predictions`` and ``annotations``. Usually, inputs are the information your model receives (for example: 'Macbeth'). 
 
 Think of predictions as the classification that your system made over that input (for example: 'Virginia Woolf'), and think of annotations as the ground truth that you manually assign to that input (because you know that, in this case, it would be 'William Shakespeare'). Records are defined by the type of ``Task``\ they are related to. Let's see three different examples:
 
-Token classification record
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Text classification record
+"""""""""""""""""""""""""""
 
-(to complete)
+Text classification deals with predicting in which categories a text fits. As if you're shown an image you could quickly tell if there's a dog or a cat in it, we build NLP models to distinguish between a Jane Austen's novel or a Charlotte Bronte's poem. It's all about feeding models with labeled examples and see how it start predicting over the very same labels.
 
 .. code-block:: python
 
-   ner_record = rb.TokenClassificationRecord(
-       annotation=[],
-     prediction=[]
+    record = rb.TextClassificationRecord(
+        inputs={
+            "text": "your input text"
+        },
+        prediction = [('label1', 0.8), ('label2', 0.2)]
+        prediction_agent = "link or reference to agent",
+
+        annotation = "label",
+        annotation_agent= "link or reference to annotator",
+
+        metadata={  # Information about this record
+            "split": "train"
+        },
+
    )
 
-Text classification record
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+Multi-label text classification record
+""""""""""""""""""""""""""""""""""""""
 
-(to complete)
+Another similar task to Text Classification, but yet a bit different, is Multilabel Text Classification. Just one key difference: more than one label may be predicted. While in a regular Text Classification task we may decide that the tweet "I can't wait to travel to Egypts and visit the pyramids" fits into the hastag #Travel, which is accurate, in Multilabel Text Classification we can classify it as more than one hastag, like #Travel #History #Africa #Sightseeing #Desert.
 
 .. code-block:: python
 
-   ner_record = rb.TextClassificationRecord(
-       annotation=[],
-     prediction=[]
+   record = rb.TextClassificationRecord(
+        inputs={
+            "text": "your input text"
+        },
+        multi_label = True,
+
+        prediction = [('label1', 0.8), ('label2', 0.2)],
```

Comment:
Change scores to reflect this is is multi label distribution with at least three labels summing more than 1.0.

In both examples use more real input.text and labels. Example: SPAM and HAM input.text "Free discounts bla bla" reading the example we should get an idea of a real model prediction.

Reply:
Should I also reference real annotators and prediction agents? I could use some HuggingFace models for prediction agents and some fake names for annotation agents.